partiql-ast/src/main/java/org/partiql/ast/ExcludeStep.java

```diff
@@ -69,6 +69,8 @@ public <R, C> R accept(@NotNull AstVisitor<R, C> visitor, C ctx) {
     @Builder(builderClassName = "Builder")
     @EqualsAndHashCode(callSuper = false)
     public static class StructWildcard extends ExcludeStep {
+        public StructWildcard() {}
```

Comment:
(self-review) public constructors were missing fro `ExcludeStep.StructWildcard` and `ExcludeStep.CollWildcard`.